src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/AdjudicationsService.kt

```diff
@@ -398,14 +398,23 @@ private fun ReportedDamageDto.Code.toNomisEnum(): RepairToUpdateOrAdd.TypeCode =
   ReportedDamageDto.Code.CLEANING -> RepairToUpdateOrAdd.TypeCode.CLEA
   ReportedDamageDto.Code.REPLACE_AN_ITEM -> RepairToUpdateOrAdd.TypeCode.DECO // best match possibly?
 }
+private fun ReportedDamageDto.Code.toNomisCreateEnum(): RepairToCreate.TypeCode = when (this) {
```

Comment:
we have 2 enum list from NOMIS API that are identical - not sure the best way to handle this other than have duplicate code for the 2 return types